x/stake/keeper/key.go

```diff
@@ -38,7 +38,7 @@ var (
 	ValidatorQueueKey    = []byte{0x43} // prefix for the timestamps in validator queue
 )
 
-const maxDigitsForAccount = 12 // ~220,000,000 atoms created at launch
+const maxDigitsForAccount = 30 // ~220,000,000 atoms created at launch
```

Comment:
`maxDigitsForAccount` is useless in cosmos-sdk v0.25.0. Changing its value doesn't make sense.